tests/foreman/api/test_discoveryrule.py

```diff
@@ -130,44 +97,67 @@ def test_positive_provision_with_rule_priority():
     :expectedresults: Host with lower count have higher priority and that
         rule should be executed first
 
-    :CaseAutomation: NotAutomated
-
     :CaseImportance: High
     """
+    discovered_host = module_target_sat.api_factory.create_discovered_host()
+
+    prio_rule = module_target_sat.api.DiscoveryRule(
+        max_count=5,
+        hostgroup=module_discovery_hostgroup,
+        search_=f'name = {discovered_host["name"]}',
+        location=[discovery_location],
+        organization=[discovery_org],
+        priority=1,
+    ).create()
 
+    rule = module_target_sat.api.DiscoveryRule(
+        max_count=5,
+        hostgroup=module_discovery_hostgroup,
+        search_=f'name = {discovered_host["name"]}',
+        location=[discovery_location],
+        organization=[discovery_org],
+        priority=10,
+    ).create()
 
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_positive_multi_provision_with_rule_limit():
-    """Create a discovery rule (CPU_COUNT = 2) with host limit 1 and
-    provision more than 2 hosts with same rule
-
-    :id: 553c8ebf-d1c1-4ac2-7948-d3664a5b450b
-
-    :Setup: Hosts with two CPUs should already be discovered
-
-    :expectedresults: Rule should only be applied to 2 discovered hosts
-        and the rule should already be skipped for the 3rd one.
-
-    :CaseAutomation: NotAutomated
+    result = module_target_sat.api.DiscoveredHost(id=discovered_host['id']).auto_provision()
+    assert f'provisioned with rule {prio_rule.name}' in result['message']
 
-    :CaseImportance: High
-    """
+    # Delete discovery rule
+    for _ in rule, prio_rule:
+        _.delete()
+        with pytest.raises(HTTPError):
+            _.read()
```

Comment:
Non blocking: any variable would be good instead of `_`

Reply:
Any way its just for teardown :)